process/types_darwin.go

```diff
@@ -50,7 +50,7 @@ package process
 
 #include <sys/sysctl.h>
 #include <sys/ucred.h>
-#include <sys/proc.h>
+#include <sys/proc).h>
```

Comment:
Yup I'll fix it. Search and replace error
On Oct 16, 2015 10:53 PM, "shirou" <notifications@github.com> wrote:

> In process/types_darwin.go
> <https://github.com/shirou/gopsutil/pull/97#discussion_r42305905>:
>
> > @@ -50,7 +50,7 @@ package process
> >
> >  #include <sys/sysctl.h>
> >  #include <sys/ucred.h>
> > -#include <sys/proc.h>
> > +#include <sys/proc).h>
>
> Is this something typo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shirou/gopsutil/pull/97/files#r42305905>.
>